packages/cli/BUILD

```diff
@@ -8,7 +8,7 @@ exports_files(["package.json"])
 
 pkg_npm(
     name = PACKAGE_NAME,
-    package_name = "@formatjs/%s" % PACKAGE_NAME,
+    package_name = "@soxhub/formatjs/%s" % PACKAGE_NAME,
```

Comment:
this doesn't match what is in package.json, likely is should be `-%s` instead of `/%s`